tasks/install-mdex.yml

```diff
@@ -3,11 +3,23 @@
   block:
     - name: "Install requirements"
       package:
-        name: "{{ package }}"
+        name: "{{ requirements }}"
         state: "present"
-      loop: "{{ requirements }}"
+      register: installed_packages
+      until: installed_packages is succeeded
+
+    - name: "Install ansible selinux configure libraries"
+      package:
+        name: "{{ selibs_item }}"
+        state: present
+      register: installed_selinux_package
+      until: installed_selinux_package is succeeded
+      loop:
```

Comment:
Loop is not needed anymore for installing packages. See "Install requirements" task above as example.

Reply:
Fixed